### ISSUE TITLE
Laggy scrolling Flutter ver 1.22

### ISSUE THREAD

**Issue by fadhlisulaimi** [google/flutter.widgets#179]
## Problem description
I am facing some lagging in scrolling after updating to flutter v1.22.
I'm not sure if i'm the only one experiencing this.
I'm pretty sure it's not about complexity of my widget tree in each row because it was never laggy before this

## Steps to reproduce
Update to flutter v1.22 in stable channel and create a simple ScrollablePositionedList.builder. Perhaps each row can be a container with a simple text widget.

**Comment by oseiasmribeiro** [comment]
Do you have StreamBuilders in the construction of your indexes?

**Comment by fadhlisulaimi** [comment]
Yes i do have, it is to update the data dynamically from firestore. I'm basically creating a chat messaging app.

**Comment by oseiasmribeiro** [comment]
Try it out, put the StreamBuilder out, or temporarily remove it.

**Comment by tarobins** [comment]
Do you see the lagginess on the example app?  If it helps, you can clone this repo to try the example: https://github.com/tarobins/spl_example_wrapper

**Comment by mwolffalonso** [comment]
Hello, I have a similar issue after moving to 1.22. Tried downgrade to 1.20.3 and works fine.
Streambuilder is not in List builder but i use it to change the position of the List.
Position goes crazy. Thanks!

**Comment by tarobins** [comment]
@mwolffalonso what  do you mean by "position goes crazy"?  Do you have to be using StreamBuilder to see the issue?
